README.md

```diff
@@ -1,194 +1,82 @@
 iDynTree [![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0) [![License: LGPL v2](https://img.shields.io/badge/License-LGPL%20v2-blue.svg)](https://www.gnu.org/licenses/lgpl-2.1)  [![ZenHub](https://img.shields.io/badge/Shipping_faster_with-ZenHub-435198.svg)](https://zenhub.com)
 ===========
 
-iDynTree is a library of robots dynamics algorithms for control, estimation and simulation.
+iDynTree is a library of robots dynamics algorithms for control, estimation and simulation. It is specifically designed for free-floating robots, but it is possible to use it also  with fixed-base robots. 
 
-iDynTree is specifically designed for free-floating robots, but it is possible to use it also  with fixed-base robots.
+The **major characteristic features** of iDynTree are:
+* It is written in **C++**, with **Python** and **MATLAB** bindings.
+* It uses an  **undirected graph data structure** (`iDynTree::Model`) that is used to represent robots, to easily change the **base link** that you are using for your kinematics and dynamics computations without the need to reload your model or change your joint or link serializations. This is done as iDynTree was developed for floating-base robots such as humanoids, in which the most convenient choice of **base link** can change.
+* It contains support for **reading and writing URDF files** from a `iDynTree::Model`, making it useful to write tools that modify robot models and saves them back to file. This is done as iDynTree was developed to developed **tools for identification of kinematics and dynamics parameters**.
```

Comment:
a typo: developed to developed

Reply:
Thanks, fixed in https://github.com/robotology/idyntree/pull/932/commits/830ad70afffd8df1b749cc04c2b78e9f01ef71a8 .